src/main/resources/locale.properties

```diff
@@ -1,45 +1,45 @@
-btnFileOpen=Select .NSP files
-btnUpload=Upload to NS
-logsEnteredAsMsg1=You have been entered as: 
-logsEnteredAsMsg2=You should be root or have configured 'udev' rules for this user to avoid any issues.
-logsFilesToUploadTitle=Files to upload:
-logsGreetingsMessage=Welcome to NS-USBloader
-logsNoFolderFileSelected=No files selected: nothing to upload.
-windowBodyConfirmExit=Data transfer is in progress and closing this application will interrupt it.\nIt's the worse thing you can do now.\nInterrupt proccess and exit?
-windowTitleConfirmExit=No, don't do this!
-btnStop=Interrupt
+btnFileOpen=Selecionar arquivos .NSP
+btnUpload=Subir para o Nintendo Switch
```

Comment:
That's not a good translation at all!!!
Should we 
"Enviar para a Nintendo switch"